sdk/src/main/java/com/mapbox/maps/MapProvider.kt

```diff
@@ -50,6 +53,12 @@ internal object MapProvider {
     return mapTelemetry
   }
 
+  fun getEventService(accessToken: String): EventsService {
+    val eventsServiceOptions =
+      EventsServerOptions(accessToken, BuildConfig.MAPBOX_EVENTS_USER_AGENT, null)
+    return EventsService.getOrCreate(eventsServiceOptions)
```

Comment:
What happens if the options are different than the ones inside the `MapTelemetryImpl` constructor?

Reply:
I can answer myself. It seems a new service will be created.

Therefore, I'm a bit concerned that we would need to maintain this `options` and the `MapTelemetryImpl` one in sync.

Reply:
then I assume we will create a new instance of `EventsService`, which is not correct. We need to make sure to use the same configuration from MapTelemetryImpl.